addons/rose/addon/styles/rose/components/table/_table.scss

```diff
@@ -35,7 +39,33 @@
     }
   }
 
+  // Grow to full available width
+  .rose-table-cell-expand {
+    width: 100%;
+
+    & > * {
+      vertical-align: inherit;
+      margin: 0;
+    }
+  }
+
   [class*='rose-button'] {
     margin-bottom: 0;
   }
+
+  &.rose-table-condensed {
+    .rose-table-cell {
+      @include type.type(s);
+      $lineHeight: 36 / 14;
```

Comment:
The type mixing can handle line height.  Is it nonstandard?

Reply:
Same note as above regarding extracting styles from key/value component for condensed table style.

Reply:
I assume, regarding the previous comment this should be fine. Otherwise let me know and I will check.